pkg/splunk/client/enterprise.go

```diff
@@ -629,10 +629,7 @@ func (c *SplunkClient) DecommissionIndexerClusterPeer(enforceCounts bool) error
 }
 
 // BundlePush pushes the CM master apps bundle to all the indexer peers
-func (c *SplunkClient) BundlePush(ignoreIdenticalBundle bool, mock bool) error {
-	if mock {
-		return nil
-	}
+func (c *SplunkClient) BundlePush(ignoreIdenticalBundle bool) error {
```

Comment:
QQ: Why did we have the mock bool in the first place if it weren't used?

Reply:
It was used Arjun. Now, with this change, the caller will not cover a UT for this case for success, and to address it holistically, I put a separate Jira cspl-581, to cover both UT and also introducing the pod manager to the CM CR.